src/main/java/sc/fiji/hdf5/HDF5_Reader_Vibez.java

```diff
@@ -156,31 +156,38 @@ public void run(String arg)
     String name = "";
     boolean tryAgain;
     String openMSG = "Open HDF5...";
-    do {
-      tryAgain = false;
-      OpenDialog od;
-      if (directory.equals(""))
-          od = new OpenDialog(openMSG, "");
-      else
-          od = new OpenDialog(openMSG, directory, "");
-      
-      directory = od.getDirectory();
-      name = od.getFileName();
-      if (name == null)
-          return;
-      if (name == "")
-          return;
-      
-      File testFile = new File(directory + name);
-      if (!testFile.exists() || !testFile.canRead())
-          return;
-      
-      if (testFile.isDirectory()) {
-        directory = directory + name;
-        tryAgain = true;
-      }
-    } while (tryAgain);
-    
+
+    if (arg != null && arg.endsWith("hdf")){
```

Comment:
Hi @lukebfunk,

`.hdf` is not the only valid extension for hdf5 files. The other options and `.h5`and `.hdf5`. Please update your code to reflect this.